.circleci/config.yml

```diff
@@ -924,17 +924,9 @@ jobs:
                 command: |
                   make devnet-up-deploy
             - run:
-                name: Check L2 config
-                command: go run cmd/check-l2/main.go --l2-rpc-url http://localhost:9545 --l1-rpc-url http://localhost:8545
-                working_directory: op-chain-ops
-            - run:
-                name: Deposit ERC20 through the bridge
-                command: timeout 8m npx hardhat deposit-erc20 --network devnetL1 --l1-contracts-json-path ../../.devnet/sdk-addresses.json
-                working_directory: packages/sdk
-            - run:
-                name: Deposit ETH through the bridge
-                command: timeout 8m npx hardhat deposit-eth --network devnetL1 --l1-contracts-json-path ../../.devnet/sdk-addresses.json
-                working_directory: packages/sdk
+                name: Test the stack
```

Comment:
This doesn't look like it's running in CI, not sure if it's a permissions issue

Reply:
I'm certainly not a circle CI expert, but it did execute in the source repo https://app.circleci.com/pipelines/github/bobanetwork/v3-anchorage/97/workflows/8f93a8dd-291f-4452-ad14-c47411905136/jobs/6089 so I believe the definition should be alright.